php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -590,16 +590,37 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                                     $pieces[2],
                                     $dateOptions
                                 );
-                            } else {
+                            }
+                            if ($dateFormat === 'BasicDate') {
                                 // Shows date without not answered dropdown
                                 $this->addBasicDate(
                                     $pieces[1],
                                     $pieces[2],
                                     $dateOptions
                                 );
                             }
+                            if ($dateFormat === 'Date') {
+                                // Shows standard date
+                                $this->addDateElement(
+                                    $pieces[1],
+                                    $pieces[2],
+                                    $dateOptions
+                                );
+                            }
                         } else {
-                            // Shows standard date
+                            // it will only enter here if the date format is not
+                            // explicitly set. This functionality is confusing and
+                            // should be deprecated
+                            error_log(
+                                "DEPRECATION MESSAGE: $filename defines the field ".
+                                "$pieces[1] as a date without specifying the ".
+                                "format explicitly. Defaulting to standard dates ".
+                                "when the format is undefined will no longer be ".
+                                "supported in future versions of loris. Make sure ".
+                                "to specify the format as `Date` in all your ".
+                                ".linst files as the 6th parameter of you date ".
+                                "entries where applicable."
```

Comment:
I don't think this description is clear enough for a user who sees it to know what action they need to take to update their instrument. In particular, I'm not sure how to interpret "Make sure to specify the format as `Date` in all your .linst files as the 6th parameter of you date entries where applicable."

Reply:
Also what's a "standard date" and why is it being removed? We haven't had any backwards incompatible changes to the LINST format and it seems unwise to start given that this is already backwards compatible.